src/complex/DataStructure/DataArray.hpp

```diff
@@ -58,16 +58,16 @@ class DataArray : public IDataArray
    *
    * +++ The created DataArray takes ownership of the provided DataStore. +++
    *
-   * @param dataStructure The parent DataStructure that will own the DataArray
+   * @param ds The parent DataStructure that will own the DataArray
    * @param name The name of the DataArray
    * @param store The IDataStore instance to use. The DataArray instance WILL TAKE OWNERSHIP of the DataStore Pointer.
    * @param parentId = {}
    * @return DataArray<T>* Instance of the DataArray object that is owned and managed by the DataStructure
    */
-  static DataArray* Create(DataStructure& dataStructure, std::string name, std::shared_ptr<store_type> store, const std::optional<IdType>& parentId = {})
+  static DataArray* Create(DataStructure& ds, std::string name, std::shared_ptr<store_type> store, const std::optional<IdType>& parentId = {})
```

Comment:
@imikejackson I honestly don't know where these changes came from because I don't recall making them.  It was most likely a bad rebase.  I'll fix it 👍